<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CConnectionDialog</class>
 <widget class="QDialog" name="CConnectionDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection properties dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="connectionNameEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>connection title (optional)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="1" column="1" colspan="7">
      <widget class="QLineEdit" name="baseEdit"/>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="versionLabel">
       <property name="text">
        <string>Version</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Base DN</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="7">
      <widget class="QLineEdit" name="userEdit">
       <property name="placeholderText">
        <string>username for connection</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QRadioButton" name="radioSimpleAuth">
       <property name="text">
        <string>Simple authentication</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QCheckBox" name="sslCheck">
       <property name="text">
        <string>SSL</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QCheckBox" name="tlsCheck">
       <property name="text">
        <string>TLS</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="7">
      <widget class="QLineEdit" name="pwdEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>password for connection</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QCheckBox" name="saslCheck">
       <property name="text">
        <string>SASL</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QRadioButton" name="radioGssAuth">
       <property name="text">
        <string>GSS-API</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="5">
      <widget class="QCheckBox" name="anonymousCheck">
       <property name="text">
        <string>Anonymous connection</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="5">
      <widget class="QLineEdit" name="hostBox"/>
     </item>
     <item row="0" column="7">
      <widget class="QSpinBox" name="portSpin">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>369</number>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QSpinBox" name="versionSpin">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>3</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Connect</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>connectionNameEdit</tabstop>
  <tabstop>hostBox</tabstop>
  <tabstop>baseEdit</tabstop>
  <tabstop>radioSimpleAuth</tabstop>
  <tabstop>sslCheck</tabstop>
  <tabstop>tlsCheck</tabstop>
  <tabstop>radioGssAuth</tabstop>
  <tabstop>saslCheck</tabstop>
  <tabstop>userEdit</tabstop>
  <tabstop>pwdEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
